litr/src/main/java/com/linkedin/android/litr/MediaTransformer.java

```diff
@@ -429,4 +444,28 @@ private MediaFormat createTargetMediaFormat(@NonNull MediaSource mediaSource,
 
         return targetMediaFormat;
     }
+
+    @Nullable
+    private MediaFormat createTargetAudioFormat(@NonNull MediaFormat sourceMediaFormat,
+                                                @Nullable MediaFormat targetAudioFormat,
+                                                @Nullable String targetMimeType) {
+        if (targetMimeType == null || targetAudioFormat != null) {
+            return targetAudioFormat;
+        }
+        MediaFormat adjustedAudioFormat = MediaFormat.createAudioFormat(
+                targetMimeType,
+                sourceMediaFormat.getInteger(MediaFormat.KEY_SAMPLE_RATE),
+                sourceMediaFormat.getInteger(MediaFormat.KEY_CHANNEL_COUNT));
+        if (sourceMediaFormat.containsKey(MediaFormat.KEY_BIT_RATE)) {
+            adjustedAudioFormat.setInteger(
+                    MediaFormat.KEY_BIT_RATE,
+                    sourceMediaFormat.getInteger(MediaFormat.KEY_BIT_RATE));
+        }
```

Comment:
Since bitrate is required further down the line, should we provide a default bitrate if it isn't set?

Reply:
Good call. Added a fallback to default audio bitrate (256 kbps)